.gitignore

```diff
@@ -8,4 +8,5 @@ yarn-error.log
 .env.*
 *.tfstate
 .vscode
-.eslintcache
\ No newline at end of file
+.eslintcache
+__recordings__/
```

Comment:
There's no other ocurrence of "__recordings__" anywhere else in this repo, can you please place a comment above this line explaining where "__recordings__" comes from?

Reply:
I added the comment, please let me know if its fine